src/presentation/index.ts

```diff
@@ -60,6 +62,13 @@ function startRundownEventServer(): void {
   EventServerFacade.createEventServer().startServer(RUNDOWN_EVENT_SERVER_PORT)
 }
 
+async function connectToDatabase(): Promise<void> {
+  const logger: Logger = LoggerFacade.createLogger().tag('startup')
+  await RepositoryFacade.getDatabase()
+    .connect()
+    .catch((reason) => logger.data(reason).error('### Failed to connect to database ###'))
```

Comment:
The `#`s seems unnecessary.

Reply:
The idea was that since this is a quite severe thing, it would be nice if it was even easier to spot in the logs.

Reply:
They have been removed